contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -268,22 +264,18 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
             blockchainID = WARP_MESSENGER.getBlockchainID();
         }
 
-        // Require that the message was intended for this blockchain and teleporter contract.
-        require(
-            warpMessage.destinationChainID == blockchainID,
-            "TeleporterMessenger: invalid destination chain ID"
-        );
-        require(
-            warpMessage.destinationAddress == address(this),
-            "TeleporterMessenger: invalid destination address"
-        );
-
         // Parse the payload of the message.
         TeleporterMessage memory teleporterMessage = abi.decode(
             warpMessage.payload,
             (TeleporterMessage)
         );
 
+        // Require that the message was intended for this blockchain and teleporter contract.
```

Comment:
It looks like we've removed a check that the `destinationAddress` is the same as `address(this)`. IIUC, this check is not needed because we already check that the message was sent by the same address as `address(this)`. If that's correct, could you just update this comment to remove the "add teleporter contract" part?

Reply:
Yes. Previously, we check that `warpMessage.destinationAddress` is same as the teleporter contract address(`address(this)`). The destinationAddress is removed from warpMessage, and we agree that we do not need to check in this [discussion](https://github.com/ava-labs/teleporter/pull/91#discussion_r1376951683). I will update the comments.